app/styles/ember-paper.scss

```diff
@@ -39,6 +39,35 @@
   padding-right: 5px;
 }
 
+// Icon spinners (not part of angular-material)
+.md-spin {
+   animation: md-spin 1.5s infinite linear;
+}
+
+.md-spin-reverse {
+  animation: md-spin-reverse 1.5s infinite linear;
```

Comment:
Better depend on the `md-spin` animation and set the `animation-direction` to reverse.